src/main/java/net/dv8tion/jda/core/entities/EntityBuilder.java

```diff
@@ -611,7 +611,8 @@ public TextChannel createTextChannel(GuildImpl guildObj, JSONObject json, long g
             .setName(json.getString("name"))
             .setTopic(json.optString("topic"))
```

Comment:
I know this is not relevant to this PR but could you change this to `optString("topic", null)`